packages/expo-updates/android/src/main/java/expo/modules/updates/loader/Loader.kt

```diff
@@ -214,7 +215,8 @@ abstract class Loader protected constructor(
     ERRORED
   }
 
-  private fun downloadAllAssets(assetList: List<AssetEntity>) {
+  private fun downloadAllAssets(update: Update) {
+    val assetList: List<AssetEntity> = update.assetEntityList
```

Comment:
optional minor nit: no need for explicit type specification in this case
```suggestion
    val assetList = update.assetEntityList
```